lib/image_server/active_record.rb

```diff
@@ -39,7 +39,7 @@ def #{column}(options={})
                       format: '#{default_format}',
                       processing: #{column}_processing?,
                       object: self }
-          ImageServer::Image.new(#{namespace_constant}, #{column}_hash, default_options.merge(options))
+          ImageServer::Image.new(#{namespace_constant}, #{column}_hash, **default_options.merge(options))
```

Comment:
Explicitly passing arguments in a hash
https://www.ruby-lang.org/en/news/2019/12/12/separation-of-positional-and-keyword-arguments-in-ruby-3-0/
See double splat entry (** thing)